arbnode/batch_poster.go

```diff
@@ -66,6 +66,14 @@ var (
 	blobGasLimitGauge             = metrics.NewRegisteredGauge("arb/batchposter/blobgas/limit", nil)
 	suggestedTipCapGauge          = metrics.NewRegisteredGauge("arb/batchposter/suggestedtipcap", nil)
 
+	batchPosterBacklogGauge = metrics.NewRegisteredGauge("arb/batchposter/backlog", nil)
```

Comment:
We already have metrics for the number of messages and bytes in the backlog. Calling this the "estimated batch backlog" or similar might be useful though. The main difference in this metric is it's divided by the estimated batch size.

Reply:
Can update the metric name!